src/emails/MagicLinkEmail.tsx

```diff
@@ -0,0 +1,105 @@
+import {
+  Body,
+  Button,
+  Container,
+  Head,
+  Heading,
+  Hr,
+  Html,
+  Link,
+  Preview,
+  Section,
+  Text,
+} from "jsx-email";
+import * as React from "react";
+
+interface MagicLinkEmailProps {
+  magicLink: string;
+}
+
+export const MagicLinkEmail = ({ magicLink }: MagicLinkEmailProps) => (
+  <Html>
+    <Head />
+    <Preview>Your magic link for OpenCap</Preview>
+    <Body style={main}>
+      <Container style={container}>
+        <Heading style={heading}>Your magic link for OpenCap</Heading>
+        <Section style={buttonContainer}>
+          <Button style={button} href={magicLink}>
+            Login
+          </Button>
+        </Section>
+        <Text style={paragraph}>
+          or copy and paste this URL into your browser:{" "}
+          <Link href={magicLink} style={reportLink}>
+            {magicLink}
+          </Link>
+        </Text>
+        <Hr style={hr} />
+        <Link href="https://opencap.co/" style={reportLink}>
```

Comment:
`"https://opencap.co/"` → `constants.url`

and

`import { constants } from "@/lib/constants";`